features/FEATURE_LWIP/lwip-interface/lwip-eth/arch/TARGET_STM/stm32xx_emac.c

```diff
@@ -46,6 +46,9 @@ static sys_mutex_t tx_lock_mutex;
 /* function */
 static void _eth_arch_rx_task(void *arg);
 static void _eth_arch_phy_task(void *arg);
+#if defined (TARGET_NUCLEO_F767ZI)
```

Comment:
As indicated in the Errata sheet, the RMII issue concerns all STM32F76xxx and TM32F77xxx,
So maybe we should increase this define test ?